src/Microsoft.Framework.ConfigurationModel.UserSecrets/PathHelper.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.Framework.Internal;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Framework.ConfigurationModel.UserSecrets
+{
+    public class PathHelper
+    {
+        private const string Secrets_File_Name = "secrets.json";
+
+        public static string GetSecretsPath([NotNull]string projectPath)
+        {
+            var projectFilePath = Path.Combine(projectPath, "project.json");
```

Comment:
@davidfowl @loudej We talked once about providing a property bag on IApplicationEnvironment that contains all the properties in the project.json, so that things like this could flow. If we did that then we wouldn't have to parse the project.json again here. I like that better than parsing this again. What do you think?